data/human/deep missions.txt

```diff
@@ -2752,3 +2752,1453 @@ mission "Stone of our Fathers 5"
+			`	"Misguided girl." Carla says sadly. "We'll take it from here. Have a wonderful day, Captain <last>.`
+				decline
+			label takeoff
+			`	You punch it, and your ship burns for the stars. Hjlod visibly relaxes when the thrust kicks in. She closes her tired eyes and slumps to the floor. "Thank Skade" is all she says.`
+				accept
+	on complete
+		payment 12481
+		event "the book" 200
+		event "nifel hjlod" 2700
+		conversation
+			scene "scene/snowvillage"
+			` Your ship touches down just outside the abandoned village of Udainskar. Silently, your passengers disembark and drift into the world they once knew and thought never to see again. The village appears to have weathered its abandonment well, and it appears the people here will not have too much trouble picking up where they left off.`
+			`	Hjlod lingers besides your ship for just a moment. "The credits are in hard-drive on your ship's console. I hope it makes you happy.`
+			`	"I do not know vhy you sent us to that place. But I am happy to be home. So go, and maybe one day I vill not hate to see you again."`
+
+
+
+event "nifel hjlod"
+
+
+
+event "the book"
+
+
+
+mission "Skadenga Call 3"
+	name "Blood Money"
+	landing
+	source
+		government "Republic"
+	destination
+		government "Republic"
+	invisible
+	to offer
+		has "Skadenga Call 2: declined"
+	on offer
+		payment 300000
+		event "asgard hjlod" 2000
+		conversation
+			`You receive a message from your bank that <payment> have been deposited into your account from the Census and Culture Bureau of The Deep. The following memo is attached:`
+			`	"For heroic and selfless service to the ideals and prosperity of The Deep."`
+				decline
+
+
+
+event "asgard hjlod"
+
+
+
+mission "Skadenga Call 4"
+	name "Hjlod's goodbye"
+	invisible
+	destination "Nifel"
+	to offer
+		has "Skadenga Call 2: done"
+		has "event: nifelhjlod"
+	on complete
+		event "hjlod waits again" 360
+		conversation
+			`A familiar woman is waiting for you outside your ship. Although it's been more than seven years since you've last seen her, you recognize Hjlod instantly. Her eyes are brighter than when last you parted, and she has the same lopsided grin that you remember. "<first>, I saw your ship landing, and thought I vould say hello."`
+			`	You ask her how she has been, and she tells you that the small community of Skadenga have rebuilt and thrived here. That there has even been a small amount of immigration to their tiny town. That word had gotten out about their desperate escape from Asgard.`
+			`	She tells you that she forgives your past actions, and hopes you find whatever it is you are looking for among the stars. She even hugs you before she leaves. Although she might forgive you, you wonder if you can ever forgive yourself...`
```

Comment:
`you wonder if you can ever forgive yourself` pushes a perspective the player themselves might not have a little too much for me. Consider finding a different way to finish this line.

Reply:
Removed the line.